--- a/src/examples/sssd.conf
+++ b/src/examples/sssd.conf
@@ -1,5 +1,4 @@
 [sssd]
-services = nss, pam
 domains = shadowutils
 
 [nss]
--- a/src/confdb/confdb.h
+++ b/src/confdb/confdb.h
@@ -46,8 +46,7 @@
 #define SSSD_LOCAL_MINID 1000
 #define CONFDB_DEFAULT_SHELL_FALLBACK "/bin/sh"
 #define CONFDB_FALLBACK_CONFIG \
-    "[sssd]\n" \
-    "services = nss\n"
+    "[sssd]\n"
 
 
 /* Configuration options */
